### PR TITLE
adding a starter kit to toolbox

### PR DESCRIPTION
**Pull request by privateducky**

### CODE REVIEW

File: plugins/toolbox/Toolbox.html [hunk 1]
```diff
@@ -150,8 +150,25 @@ <h3 id="tool-name" class="split"><span class="main"></span></h3>
...
     function lastUpdated(name){
         exec(`git log -n1 --pretty='format:%cd' --date=format:'%Y-%m-%d'`, {cwd: path.join(TOOLBOX, name)}, (error, stdout, stderr) => {
-            console.log(stdout)
             $('#tool-'+name).find('#tool-url').text(`Last updated: ${stdout}`);
         })
     }
+
+    function installStarterKit(){
+        const kit = [
+            'https://github.com/preludeorg/pneuma',
```
Comment (privateducky) [inline plugins/toolbox/Toolbox.html:159]: @khyberspache @bfuzzy1 what do y'all think of these? literally just pulled out of my own local toolbox but it'd be nice to have a 5-10 tool starter kit of tools specific to parts of the red team process/phases.

Comment (khyberspache) [reply plugins/toolbox/Toolbox.html:159]: oooooooooooo lemme put together my list and I'll post it here

Comment (bfuzzy1) [reply plugins/toolbox/Toolbox.html:159]: Putting together mine as well!

Comment (bfuzzy1) [reply plugins/toolbox/Toolbox.html:159]: Nice list to get started with:

https://github.com/sherlock-project/sherlock.git
https://github.com/laramies/theHarvester.git
https://github.com/mdsecactivebreach/SharpPack.git
https://github.com/darryllane/Bluto.git
https://github.com/byt3bl33d3r/CrackMapExec.git
https://github.com/byt3bl33d3r/SprayingToolkit.git
https://github.com/sevagas/macro_pack.git
https://github.com/emtunc/SlackPirate.git

Comment (privateducky) [reply plugins/toolbox/Toolbox.html:159]: so what'dya think... select 2 tools per "category" to start with?... maybe categories named after ATT&CK tactics? maybe phases of an operation?

Comment (bfuzzy1) [reply plugins/toolbox/Toolbox.html:159]: Yea, I think that would work. ATT&CK tactic would be good for sure as it's a pretty commonly known thing at this point across orgs.

**Pull request merged**